src/reducers/CoursePlannerReducer.jsx

```diff
@@ -16,21 +16,247 @@ const initialState = {
       category: 'ME Math',
       courses: [
         {
-          name: 'Partial Differential Equations',
-          courseId: 'MTH3120',
+          courseCode: 'MTH3120',
+          title: 'Partial Differential Equations',
         },
         {
-          name: 'Numerical Methods and Scientific Computing',
-          courseId: 'MTH3150',
+          courseCode: 'MTH3150',
+          title: 'Numerical Methods and Scientific Computing',
         },
         {
-          name: 'Nonlinear Dynamics and Chaos',
-          courseId: 'MTH3170',
+          courseCode: 'MTH3170',
+          title: 'Nonlinear Dynamics and Chaos',
         },
       ],
       selected: ' ',
     },
   ],
+  allCourses: [
```

Comment:
Yeah, that's the next order of business for me. Those are just pulled from the LoginReducer (at the time, that's where they were) except I was having issues with it so I just copy-pasted